### PR TITLE
Modify our docs to use more inclusive language

### PR DESCRIPTION
**Pull request by nickvigilante**

### CODE REVIEW

**Review by lnhsingh — COMMENTED** [review]
Nice! LGTM

> *[Reviewable](https://reviewable.io/reviews/cockroachdb/docs/13902#-:-N3yPMnMA87OFDZsdznF:bcyhil1)* status: :shipit: complete! 0 of 0 LGTMs obtained (waiting on @andf-crl)

File: v21.2/sql-tuning-with-explain.md [hunk 1]
```diff
@@ -5,7 +5,7 @@ toc: true
 docs_area: develop
 ---
 
-This tutorial walks you through the common reasons for [slow SQL statements](query-behavior-troubleshooting.html#identify-slow-statements) and describes how to use [`EXPLAIN`](explain.html) to troubleshoot the issues in queries against the [`movr` example dataset](cockroach-demo.html#datasets).
+This tutorial guides you through the common reasons for [slow SQL statements](query-behavior-troubleshooting.html#identify-slow-statements) and describes how to use [`EXPLAIN`](explain.html) to troubleshoot the issues in queries against the [`movr` example dataset](cockroach-demo.html#datasets).
```
Comment (andf-crl) [inline v21.2/sql-tuning-with-explain.md:8]: nit: one cannot be guided through reasons. Suggest: `presents` instead.

File: v22.1/orchestrate-a-local-cluster-with-kubernetes-insecure.md [hunk 2]
```diff
@@ -12,7 +12,7 @@ docs_area: deploy
...
 
-This page walks you through a simple demonstration, using the open-source [Kubernetes](http://kubernetes.io/) orchestration system. Using either the CockroachDB [Helm](https://helm.sh/) chart or a few configuration files, you'll quickly create a 3-node local cluster. You'll run some SQL commands against the cluster and then simulate node failure, watching how Kubernetes auto-restarts without the need for any manual intervention. You'll then scale the cluster with a single command before shutting the cluster down, again with a single command.
+This page guides you through a simple demonstration, using the open-source [Kubernetes](http://kubernetes.io/) orchestration system. Using either the CockroachDB [Helm](https://helm.sh/) chart or a few configuration files, you'll quickly create a 3-node local cluster. You'll run some SQL commands against the cluster and then simulate node failure, watching how Kubernetes auto-restarts without the need for any manual intervention. You'll then scale the cluster with a single command before shutting the cluster down, again with a single command.
```
Comment (andf-crl) [inline v22.1/orchestrate-a-local-cluster-with-kubernetes-insecure.md:15]: nit: we do not need both `guides you though` and `a simple demonstration`: they are the same. `demonstration` is also presented without context: a demo of what? How about:

> This page demonstrates a basic integration with the open-source [Kubernetes](http://kubernetes.io/) orchestration system.

File: v22.1/orchestrate-a-local-cluster-with-kubernetes.md [hunk 3]
```diff
@@ -13,7 +13,7 @@ docs_area: deploy
...
 
-This page walks you through a simple demonstration, using the open-source [Kubernetes](http://kubernetes.io/) orchestration system. Using either the CockroachDB [Helm](https://helm.sh/) chart or a few configuration files, you'll quickly create a 3-node local cluster. You'll run some SQL commands against the cluster and then simulate node failure, watching how Kubernetes auto-restarts without the need for any manual intervention. You'll then scale the cluster with a single command before shutting the cluster down, again with a single command.
+This page guides you through a simple demonstration, using the open-source [Kubernetes](http://kubernetes.io/) orchestration system. Using either the CockroachDB [Helm](https://helm.sh/) chart or a few configuration files, you'll quickly create a 3-node local cluster. You'll run some SQL commands against the cluster and then simulate node failure, watching how Kubernetes auto-restarts without the need for any manual intervention. You'll then scale the cluster with a single command before shutting the cluster down, again with a single command.
```
Comment (andf-crl) [inline v22.1/orchestrate-a-local-cluster-with-kubernetes.md:16]: Same nit: suggest `demonstrates a basic integration with`

File: v22.1/security-reference/config-secure-hba.md [hunk 4]
```diff
@@ -10,7 +10,7 @@ CockroachDB allows fine-grained configuration of which database connect attempts
 - **Who** is making the attempt (SQL user).
 - **Where** on the internet (IP Address) the attempt is coming from.
 
-This document describes the rationale for restricting database access to specific IP ranges as a security measure, and then walks through the procedure using [authentication configuration](authentication.html) to achieve that aim.
+This document describes the rationale for restricting database access to specific IP ranges as a security measure, and then describes the procedure using [authentication configuration](authentication.html) to achieve that aim.
```
Comment (andf-crl) [inline v22.1/security-reference/config-secure-hba.md:13]: Same nit: suggest `demonstrates` for the second `describes`

**Review by andf-crl — APPROVED** [review]
LGTM with some minor nits and suggestions. Thanks!

File: v21.2/encryption.md [hunk 5]
```diff
@@ -105,4 +105,4 @@ To check that the new key is active, use the stores report page in the DB Consol
 To disable encryption, specify `key=plain`. The data keys will be stored in plaintext and new data will not be encrypted.
 
 To rotate keys, specify `key=/path/to/my/new-aes-128.key` and `old-key=/path/to/my/old-aes-128.key`. The data keys
-will be decrypted using the old key and then encrypted using the new key. A new data key will also be generated.
\ No newline at end of file
+will be decrypted using the old key and then encrypted using the new key. A new data key will also be generated.
```
[the remark this answers is not in the document]
Comment (nickvigilante) [reply v21.2/encryption.md:108]: I don't know either, but this document also has a lot of inconsistent line breaks in it in between sentences, so I fixed those for the sake of fixing it.

**Pull request merged**